src/label/creator.py

```diff
@@ -26,7 +26,7 @@
 JSON_VERSION_KEY = 'v'
 
 TEMP_STORAGE_LENGTH = 30
```

Comment:
Isn't temporary storage allowed to be lying around for 90 days?

```suggestion
TEMP_STORAGE_LENGTH = 90
```

Reply:
We discussed this and agreed to lower this limit, but I think to 45.

Reply:
![image](https://user-images.githubusercontent.com/4609724/76707268-f613da80-66ed-11ea-9c64-9f6d1781cd4e.png)

Fixed label according to instruction.